build/copyright/ccri-2016.txt

```diff
@@ -0,0 +1,5 @@
+Copyright (c) ${project.inceptionYear}-2016 ${owner}
```

Comment:
I had to keep 2016 in there unless we go through and update every single file to have 2017...